tests/exclude_system_edl/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+# This test is an example of how to write an OE application with explicit
+# ecall/ocall opt-in as detailed in <path to design document>.
+if (NOT COMPILE_SYSTEM_EDL)
+    add_subdirectory(host)
+
+    if (BUILD_ENCLAVES)
+        add_subdirectory(enc)
+    endif()
+
+    add_enclave_test(tests/exclude_system_edl
+                     exclude_system_edl_host exclude_system_edl_enc)
+else ()
+    # Throw a helpful error here
```

Comment:
Is this a TODO or what?

Reply:
Oh I forgot to change this. I put the check at the root cmakelists.txt instead. Will remove